File: CPP/BEE-1044_Multiplos.cpp
/*-------------------*
| Rodrigo CavanhaMan |
| BEE 1044           |
| Multiplos          |
*--------------------*/
#include<stdio.h>
using namespace std;

int main() {

	int a,b;

	scanf("%d",&a);
	scanf("%d",&b);

	if (a%b==0 || b%a == 0)
		printf("Sao Multiplos\n");
	else
		printf("Nao sao Multiplos\n");

	return 0;
}

File: Python/BEE-1044_Multiplos.py
/*
+--------------------+
¦ Rodrigo CavanhaMan ¦
¦        IFTM        ¦
¦      BEE 1003      ¦
+--------------------+
*/
# -*- coding: utf-8 -*-

A,B = map(int,input().split())

if A%B == 0 or B%A == 0:
    print("Sao Multiplos")
else:
    print("Nao sao Multiplos")